packages/wrangler/.eslintrc.js

```diff
@@ -8,11 +8,6 @@ module.exports = {
 		"templates",
 		"emitted-types",
 	],
-	settings: {
```

Comment:
also move this to the base react eslint config so we don't have to write this everywhere